base/loading.jl

```diff
@@ -1158,7 +1160,7 @@ function set_pkgorigin_version_path(pkg, path)
             d = parsed_toml(project_file)
             v = get(d, "version", nothing)
             if v !== nothing
-                pkgorigin.version = VersionNumber(v)
+                pkgorigin.version = VersionNumber(v)   # can we narrow the types on `v`?
```

Comment:
It has to be a string (for this not to error)

Reply:
Thanks. Just verifying, it's guaranteed it can't be a tuple? https://github.com/JuliaLang/julia/blob/ee6115bbb2c2e8d42cb878ab0aef34eebac36435/base/version.jl#L82 is what I was worried about.

Reply:
Yes, you cannot get a tuple out of TOML.